maintenance/tables.sql

```diff
@@ -1098,10 +1098,7 @@ CREATE TABLE /*_*/interwiki (
   -- insertion.
   iw_url blob NOT NULL,
 
-  -- The URL of the file api.php
-  iw_api blob NOT NULL,
-
-  -- The name of the database (for a connection to be established with wfGetLB( 'wikiid' ))
```

Comment:
This column is said to be used by MediaWiki 1.17+ - https://www.mediawiki.org/wiki/Manual:Interwiki_table#iw_api

> This is not used except by ApiQuerySiteinfo::appendInterwikiMap(),